src/server/db/mongo/user-privat-provider.js

```diff
@@ -232,7 +232,17 @@ class UserPrivate {
   }
 
   /**
-   * Fetches tasks of the type specifid with optional filtering and locks them by setting running status
+   * cancel queued task
+   * @param {string} taskName
+   * @param {object} filters
+   */
+  async cancelTasksQueued(taskName: string, filters: object = {}): Promise<void> {
+    const { taskModel } = this
+    await taskModel.deleteOne({ ...filters, taskName })
```

Comment:
```suggestion

    await taskModel.deleteOne({ ...filters, taskName })
```

Reply:
is it just me or did you change nothing in this suggestion? am I missing something?

Reply:
is suggested to addd just  an empty line before (to make code more readable)